app/client/components/PageNotFound.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+class PageNotFound extends React.Component {
```

Comment:
So according to the Javascript Style guide used in Andela , you are supposed to use a pure function since this is a stateless react component.